scripts/gitlab/gitlab-ci.yml

```diff
@@ -0,0 +1,31 @@
+stages:
+  - build
+  - test
+  - deploy
+
+variables:
+  APP_NAME: Project
+
+before_script:
+  - echo "hello world"
+
+build:
+  stage: build
+  script:
+    - composer install
+    - blt setup:build
+
+validate:
+  stage: test
+  script:
+    - blt validate
+
```

Comment:
Seems like we should call `blt tests` here with some arguments like `--define tests.run-server=true`